test/functional/helpers.lua

```diff
@@ -614,7 +614,7 @@ local function new_pipename()
 end
 
 local function missing_provider(provider)
-  if provider == 'ruby' then
+  if provider == 'ruby' or provider == 'node' then
```

Comment:
Just a general note: the nodejs host is not really a "provider", it's just a "host". I missed this when I merged that PR. 

Providers are shims for features that otherwise would be implemented in C (such as `:ruby`, `:python`, clipboard).